bittensor/_proto/bittensor.proto

```diff
@@ -92,6 +92,10 @@ message TensorMessage {
 	// Synapses hold function information this tells the axon how to use the tensor inputs.
 	// i.e. where to send them
  	repeated Synapse synapses = 9;
+
+	// Message Signature generated as: wallet.hotkey.keypair.sign( wallet.hotkey.keypair.ss58_address + str(TensorMessage.tensors[0]) + str(TensorMessage.tensors[1]) + ...  )
+	// Receivers check vis a via: keypair( TensorMessage.hotkey ).verify( TensorMessage.hotkey + str(TensorMessage.tensors[0]) + str(TensorMessage.tensors[1]) + ... , TensorMessage.signature )
+ 	string signature = 10;
```

Comment:
Is this built yet? Also, will this allow for backward compatibility with previous versions?